src/careamics/prediction_utils/stitch_prediction.py

```diff
@@ -9,6 +9,79 @@
 from careamics.config.tile_information import TileInformation
 
 
+class TilingMode:
+    """
+    Enum for the tiling mode.
+    """
+
+    TrimBoundary = 0
+    PadBoundary = 1
+    ShiftBoundary = 2
+
+
+def stitch_predictions_new(predictions, dset):
```

Comment:
It should have a more informative name, such as `stitch_prediction_microsplit` or `stitch_prediction_datamanager`.

Reply:
I don't see the change